content/kotlin/concepts/datetime/terms/offsetIn/offsetIn.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.offsetIn()'
+Description: 'Provides the time zone offset at a specific point in time.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **offsetIn()** function is used to obtain the time zone offset at a specific point in time. It provides the difference between the local time and UTC for a given moment.
+
+## Syntax
+
+```pseudo
+fun Instant.offsetIn(timeZone: TimeZone): UtcOffset
+```
+
+- `Instant`: a moment in time.
+- `utcOffset`: An offset from UTC (Coordinated Universal Time).
+- `TimeZone`: provides a conversion between `Instant` and `LocalDateTime` values using a collection of rules.
+
+## Example
+
+```kotlin
+import kotlinx.datetime.TimeZone
+
+fun main() {
+  val currentTime = Instant.now()
+  val timeZone = TimeZone.of("Asia/Kolkata")
+
+  val offset = currentTime.offsetIn(timeZone)
+
+  println("Offset from UTC in Kolkata is $offset minutes.")
+}
```

Comment:
```suggestion
import kotlinx.datetime.TimeZone

fun main() {
  val currentTime = Instant.now()
  val timeZone = TimeZone.of("Asia/Kolkata")

  val offset = currentTime.offsetIn(timeZone)

  println("Offset from UTC in Kolkata is $offset minutes.")
}
```
The now() function was deprecated in Kotlin Datetime 0.3.0 so use any alternative function.

Reply:
Sure I'll do it soon! And what to do with periodUntil.md file now it somehow get commited by me

Reply:
@vrun1208 Delete periodUntil.md file from the branch then commit and push the changes.

Reply:
@avdhoottt, Clock.System.now() would be an alternative for now() by importing kotlinx.datetime.Clock

Reply:
@vrun1208 You can use it then..